handleCustomErrors = (error, request, response, next) => {

  next(error);
 
};

module.exports = handleCustomErrors


